themes/gatsby-theme-blog-mdx/src/components/layout.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import { useTypography, TypographyProvider } from "typography-system"
+
+export default ({ children }) => {
+  const typography = useTypography()
+
+  return (
+    <TypographyProvider theme={typography}>
```

Comment:
Isn't this context provided with the `MDXThemeProvider`? I also feel like generally `typography-system` is pretty unstable at the moment. It currently uses React's `useEffect` to add the Google font for example

Reply:
Just pushed a commit that removes this file entirely since it's no longer being used. Should I opt for `@styled-system/typography` in this theme? I'd like to steer towards something more composition-based to ensure as much flexibility as possible.